spec/acceptance/default_spec.rb

```diff
@@ -12,18 +12,18 @@
             class { '::telegraf':
               hostname  => 'test.vagrant.dev',
               outputs   => {
-                  'influxdb' => {
+                  'influxdb' => [{
                     'urls'     => [ 'http://localhost:8086' ],
                     'database' => 'telegraf',
                     'username' => 'telegraf',
                     'password' => 'metricsmetricsmetrics',
-                  }
+                  }]
               },
               inputs    => {
-                  'cpu' => {
+                  'cpu' => [{
```

Comment:
This same change needs to be made in manifests/params.pp defaults. (sorry for being so late with this comment) @hanej @nrdmn